apps/tx-builder/src/components/Builder.tsx

```diff
@@ -143,6 +152,11 @@ export const Builder = ({
       return;
     }
 
+    if (!isInputValueValid(valueInput)) {
+      setValueError(`${nativeCurrencySymbol} value`);
+      return;
+    }
+
```

Comment:
Wouldn't it make more sense to disable the button?

Shouldn't it be already showing the error value, that should be set in the function `onValueInputChange`?

Reply:
I think is better to let the user make errors by pressing buttons. Disabling buttons i think is a little frustrating from a user perspective because sometimes you don't know what to do.

Besides that, If we would like to go that way perhaps we should do the same with the other form fields right? ... for keeping consistency

Reply:
Not a strong opinion there, we can get over this again if necessary when we have design updates